src/components/books.js

```diff
@@ -1,33 +1,61 @@
-import React from 'react';
-import { useSelector, useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/booksSlice';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { removeBook, getBooks } from '../redux/books/booksSlice';
 import '../App.css';
 import AddNewBook from './addNewBook';
 
 const Books = () => {
-  const theBooks = useSelector((state) => state.books);
+  const { isLoading } = useSelector((state) => state.books);
+  const theBooks = useSelector((state) => state.books.books);
+
   const dispatch = useDispatch();
 
+  const handleRemoveBook = (id) => {
+    dispatch(removeBook(id));
+  };
+  useEffect(() => {
+    dispatch(getBooks());
+  }, [dispatch]);
+
+  if (isLoading) {
+    return (
+      <h1>Loading...</h1>
+    );
+  }
```

Comment:
- [Optional] Please make sure that the user can remove books in real-time for now the remove functionality works only when the user press remove and then reload the page
![image](https://user-images.githubusercontent.com/74744787/230733989-6ddc0da1-bbfe-454e-963c-7df709504753.png)